printlabels/printlabels.js

```diff
@@ -121,19 +123,28 @@ function updateSize() {
 }
 
 ready(function() {
+  grist.ready({
+    requiredAccess: 'read table'
+  });
+  // Listen to configuration change.
+  grist.onOptions(options => {
+    data.template = options?.template ? findTemplate(options.template) : defaultTemplate;
```

Comment:
Shouldn't this fall back to `startTemplate`? Just checked that a doc with `#labels10` ignores that URL fragment, but changing defaultTemplate -> startTemplate here fixes that.

Reply:
Yes, sorry, last-minute refactor. I simplified this logic.